.github/workflows/unit-test-ci.yml

```diff
@@ -43,6 +43,8 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Set up dependencies
-        run: make setup
+        run: make setup install
+      - name: Lint codes
+        run: make lint
```

Comment:
Better to move lint to a separate workflow

Reply:
done